<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeViewer</class>
 <widget class="QDialog" name="TimeViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labe">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="msecLineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Milliseconds:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minutes:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minutesLineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
